deployment/cdk/opensearch-service-migration/test/common-utilities.test.ts

```diff
@@ -0,0 +1,27 @@
+import {CpuArchitecture} from "aws-cdk-lib/aws-ecs";
+import {validateFargateCpuArch} from "../lib/common-utilities";
+
+test('Test valid fargate cpu arch strings can be parsed', () => {
+    const cpuArch1 = "arm64"
+    const detectedArch1 = validateFargateCpuArch(cpuArch1)
+    expect(detectedArch1).toEqual(CpuArchitecture.ARM64)
+
+    const cpuArch2 = "ARM64"
+    const detectedArch2 = validateFargateCpuArch(cpuArch2)
+    expect(detectedArch2).toEqual(CpuArchitecture.ARM64)
+
+    const cpuArch3 = "x86_64"
+    const detectedArch3 = validateFargateCpuArch(cpuArch3)
+    expect(detectedArch3).toEqual(CpuArchitecture.X86_64)
+
+    const cpuArch4 = "X86_64"
+    const detectedArch4 = validateFargateCpuArch(cpuArch4)
+    expect(detectedArch4).toEqual(CpuArchitecture.X86_64)
```

Comment:
Can we have one test that passes nothing and expects it to be something valid? I think that will currently fail on X86 as my test with node showed the response to process.arch is `x64`

Reply:
Added test case 👍